docs/content/docs/custom-resource/autoscaler.md

```diff
@@ -286,16 +286,20 @@ please download JDBC driver and initialize database and table first.
 
 ```
 JDBC_DRIVER_JAR=./mysql-connector-java-8.0.30.jar
-# export the password of jdbc state store
+# export the password of jdbc state store & jdbc event handler
 export STATE_STORE_JDBC_PWD=123456
+export EVENT_HANDLER_JDBC_PWD=123456
 
 java -cp flink-autoscaler-standalone-{{< version >}}.jar:${JDBC_DRIVER_JAR} \
 org.apache.flink.autoscaler.standalone.StandaloneAutoscalerEntrypoint \
 --autoscaler.standalone.fetcher.flink-cluster.host localhost \
 --autoscaler.standalone.fetcher.flink-cluster.port 8081 \
 --autoscaler.standalone.state-store.type jdbc \
 --autoscaler.standalone.state-store.jdbc.url jdbc:mysql://localhost:3306/flink_autoscaler \
---autoscaler.standalone.state-store.jdbc.username root
+--autoscaler.standalone.state-store.jdbc.username root \
+--autoscaler.standalone.event-handler.type jdbc \
+--autoscaler.standalone.event-handler.jdbc.url jdbc:mysql://localhost:3306/flink_autoscaler \
+--autoscaler.standalone.event-handler.jdbc.username root
```

Comment:
These database related options are similar between jdbcStateStore and jdbcEventHandler. I'm not sure whether we should merge them or stay as is.

Merge them means we rename `autoscaler.standalone.state-store.jdbc.username` to `autoscaler.standalone.jdbc-plugin.username`. And both of jdbcStateStore and jdbcEventHandler will use `autoscaler.standalone.jdbc-plugin.username` and `autoscaler.standalone.jdbc-plugin.url`.

If  we define `autoscaler.standalone.event-handler.jdbc.url` and `autoscaler.standalone.state-store.jdbc.url`, it means jdbcStateStore and jdbcEventHandler can use different databases and different users.

WDYT? @gyfora  @mxm

Reply:
I'm still working on this PR now, I found it's better to include this PR in `1.8.0`. Because I believe  jdbcStateStore and jdbcEventHandler  always work together.

During I test the `jdbcEventHandler`, I found some defects of `jdbcStateStore`, such as: FLINK-34389, this comment https://github.com/apache/flink-kubernetes-operator/pull/765#discussion_r1480012807. 

It means if we includes `jdbcEventHandler` in  this version we can ensure they work well together. 

So I'd like to increase this priority, WDYT?

Note: the prod code is done(you can help review if you are free), and the progress of unit test and ITCase is about 40% and I will finish it asap..

Reply:
@1996fanrui I agree that we should probably merge the configs as they will be used together. Instead of calling it `jdbc-plugin` we could just call the properties:
```
autoscaler.standalone.jdbc.username
autoscaler.standalone.jdbc.url
...

```

Reply:
Thanks for the feedback! I have updated.